lib/widget/app_scaffold.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+
+class AppScaffold extends StatelessWidget {
+  const AppScaffold({
```

Comment:
no need for this class as well 🙂

Reply:
Hello sir, I've used it twice on the homepage,  the other one is for the circular progression assuming there will be a problem on the api.  are there any more justifications for its single use?  Thank you. :)

Reply:
I appreciate the loading part u made, but that's not necessary atm since that would be my next topic next week 🙂